brainstormforce/astra
# fix: Image , Color, & Gradient compatibility with each other in Background CSS for new control.

### Description
 - Trello task - https://trello.com/c/ImTWwmnr/263-image-color-gradient-compatibility-with-each-other-in-background-css-for-new-control

## Types of changes
Bugfix (non-breaking change which fixes an issue)

### How has this been tested?
 - Check if the Color applies like an overlay to the Image if both are set.
 - Gradient is independent and not overlay on the image. ( for now ).
 - If we set Image through desktop and add color through tablet then the desktop image should be visible with tablet color overlay.
 - If we set Image through desktop and also through tablet add color through mobile then the tablet image should be visible with mobile color overlay.
- Check if different combinations.

### Checklist:
